app/cypress/integration/cif/project-revision/index.spec.js

```diff
@@ -191,10 +195,18 @@ describe("the new project page", () => {
     cy.mockLogin("cif_admin");
 
     cy.visit("/cif/projects");
+    cy.get("button").contains("Add a Project").click();
+
+    // NEW
+    cy.url().should("include", "/new");
+    cy.findByRole("button", { name: /^confirm/i }).click();
+    cy.happoAndAxe("Project New Form", "with errors", ".error-detail", true);
+    cy.fillAndCheckNewProjectForm("Emissions Performance", "2020");
+    cy.findByRole("button", { name: /^confirm/i }).click();
 
     // OVERVIEW
-    cy.get("button").contains("Add a Project").click();
     cy.url().should("include", "/form/0");
+    cy.findByText("Emissions Performance").should("be.visible");
```

Comment:
In the happo screenshots, the RFP info isn't filled out on the overview page. It works locally so I think it's a race conditions thing. I'm forcing it to check that the info is visible but it still doesn't doesn't show it in the screenshots. Thoughts?

Reply:
It doesn't seem like this is a unique problem to this PR. The following test passes and cypress is able to find all options in the dropdown selector by text. 

```  
it.only("selects funding stream and year", () => {
    cy.mockLogin("cif_admin");
    cy.visit("/cif/projects");
    cy.get("button").contains("Add a Project").click();
    cy.url().should("include", "/new");
    cy.happoAndAxe("Project New Form", "unfilled", "main");
    cy.fillAndCheckNewProjectForm("Emissions Performance", "2020");
    cy.findByText("Emissions Performance").should("be.visible");
    cy.findByText("Select a Funding Stream").should("be.visible");
    cy.findByText("Innovation Accelerator").should("be.visible");
  });
```

I was also not seeing the correct dropdown selected in the DOM snapshots when running these tests locally

Reply:
Know cypress issue, made a ticket to check if they've fixed it in the future: https://github.com/bcgov/cas-cif/issues/1298